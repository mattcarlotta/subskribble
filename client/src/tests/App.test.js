import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import App from '../routes';

describe('[APP]', () => {
  it('should render without crashing', () => {
    shallow(<App />);
  });

  it('should match a snapshot', () => {
    const wrapper = shallow(<App />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  })
})
