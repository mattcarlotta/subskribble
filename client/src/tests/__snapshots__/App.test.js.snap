// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[APP] should match a snapshot 1`] = `
<LocaleProvider
  locale={
    Object {
      "Calendar": Object {
        "lang": Object {
          "backToToday": "Back to today",
          "clear": "Clear",
          "dateFormat": "M/D/YYYY",
          "dateSelect": "Select date",
          "dateTimeFormat": "M/D/YYYY HH:mm:ss",
          "dayFormat": "D",
          "decadeSelect": "Choose a decade",
          "month": "Month",
          "monthBeforeYear": true,
          "monthSelect": "Choose a month",
          "nextCentury": "Next century",
          "nextDecade": "Next decade",
          "nextMonth": "Next month (PageDown)",
          "nextYear": "Next year (Control + right)",
          "now": "Now",
          "ok": "Ok",
          "placeholder": "Select date",
          "previousCentury": "Last century",
          "previousDecade": "Last decade",
          "previousMonth": "Previous month (PageUp)",
          "previousYear": "Last year (Control + left)",
          "rangePlaceholder": Array [
            "Start date",
            "End date",
          ],
          "timeSelect": "Select time",
          "today": "Today",
          "year": "Year",
          "yearFormat": "YYYY",
          "yearSelect": "Choose a year",
        },
        "timePickerLocale": Object {
          "placeholder": "Select time",
        },
      },
      "DatePicker": Object {
        "lang": Object {
          "backToToday": "Back to today",
          "clear": "Clear",
          "dateFormat": "M/D/YYYY",
          "dateSelect": "Select date",
          "dateTimeFormat": "M/D/YYYY HH:mm:ss",
          "dayFormat": "D",
          "decadeSelect": "Choose a decade",
          "month": "Month",
          "monthBeforeYear": true,
          "monthSelect": "Choose a month",
          "nextCentury": "Next century",
          "nextDecade": "Next decade",
          "nextMonth": "Next month (PageDown)",
          "nextYear": "Next year (Control + right)",
          "now": "Now",
          "ok": "Ok",
          "placeholder": "Select date",
          "previousCentury": "Last century",
          "previousDecade": "Last decade",
          "previousMonth": "Previous month (PageUp)",
          "previousYear": "Last year (Control + left)",
          "rangePlaceholder": Array [
            "Start date",
            "End date",
          ],
          "timeSelect": "Select time",
          "today": "Today",
          "year": "Year",
          "yearFormat": "YYYY",
          "yearSelect": "Choose a year",
        },
        "timePickerLocale": Object {
          "placeholder": "Select time",
        },
      },
      "Modal": Object {
        "cancelText": "Cancel",
        "justOkText": "OK",
        "okText": "OK",
      },
      "Pagination": Object {
        "items_per_page": "/ page",
        "jump_to": "Goto",
        "jump_to_confirm": "confirm",
        "next_3": "Next 3 Pages",
        "next_5": "Next 5 Pages",
        "next_page": "Next Page",
        "page": "",
        "prev_3": "Previous 3 Pages",
        "prev_5": "Previous 5 Pages",
        "prev_page": "Previous Page",
      },
      "Popconfirm": Object {
        "cancelText": "Cancel",
        "okText": "OK",
      },
      "Select": Object {
        "notFoundContent": "Not Found",
      },
      "Table": Object {
        "emptyText": "No data",
        "filterConfirm": "OK",
        "filterReset": "Reset",
        "filterTitle": "Filter menu",
        "selectAll": "Select current page",
        "selectInvert": "Invert current page",
      },
      "TimePicker": Object {
        "placeholder": "Select time",
      },
      "Transfer": Object {
        "itemUnit": "item",
        "itemsUnit": "items",
        "notFoundContent": "Not Found",
        "searchPlaceholder": "Search here",
        "titles": Array [
          "",
          "",
        ],
      },
      "Upload": Object {
        "previewFile": "Preview file",
        "removeFile": "Remove file",
        "uploadError": "Upload error",
        "uploading": "Uploading...",
      },
      "locale": "en",
    }
  }
>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Router
      history={
        Object {
          "createHref": [Function],
          "createKey": [Function],
          "createLocation": [Function],
          "createPath": [Function],
          "getCurrentLocation": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "listen": [Function],
          "listenBefore": [Function],
          "push": [Function],
          "replace": [Function],
          "transitionTo": [Function],
          "unsubscribe": [Function],
        }
      }
      onUpdate={[Function]}
      render={[Function]}
      routes={
        <Unknown>
          <Redirect
            from="/"
            to="/subskribble"
          />
          <Route
            component={[Function]}
            path="/subskribble"
          >
            <IndexRoute
              component={[Function]}
            />
            <Route
              component={[Function]}
              path="subscribers"
            />
          </Route>
          <Route
            component={[Function]}
            path="*"
          />
        </Unknown>
      }
    />
  </Provider>
</LocaleProvider>
`;
